docs/source/projects/architecture/content-architecture.rst

```diff
@@ -49,6 +49,10 @@ between the frontends and the IPython Kernel is described in
 The core execution machinery for the kernel is shared with terminal IPython:
```

Comment:
Question about a probably very minor issue: is the fact that there is a colon at the end of the sentence somehow conveyed by the screen reader? In which case, would the document flow for the screen reader user be somewhat disrupted if the phrase preceding the colon is not semantically connected to the phrase following the colon?

In other words, I'm concerned that to a screen reader, we have

> The core execution machinery for the kernel is shared with terminal IPython:
> A kernel process can be connected to more than one frontend simultaneously. In
> this case, the different frontends will have access to the same variables.

But it doesn't make sense to join those two statements with a colon. It makes sense to join-via-colon the first sentence to the image that it describes/captions.

This colon-terminated phrase is really a caption for the image. But the whole figure-caption machinery as of now creates a bad user experience. So I wonder what a good solution would be.

Reply:
Is there a reason not to change it to a period? Maybe I'm missing something.